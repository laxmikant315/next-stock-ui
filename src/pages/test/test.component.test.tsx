import React from 'react';
import { shallow, mount } from 'enzyme';
import Login from './test.component';


describe('<Login /> with no props', () => {

    const state= {
        email: 'acesmndr@gmail.com',
        password: 'notapassword',
        isLoginDisabled:false
      }


    
  // jest.spyOn(React, 'useState').mockImplementation(()=>[state,()=>{}]);

  const container = mount(<Login {...state} />);
  

  it('should match the snapshot', () => {
    expect(container).toMatchSnapshot();
  });

 
});