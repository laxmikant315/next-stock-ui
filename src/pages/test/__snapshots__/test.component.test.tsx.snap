// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> with no props should match the snapshot 1`] = `
<Login
  email="acesmndr@gmail.com"
  isLoginDisabled={false}
  password="notapassword"
>
  <form>
    "acesmndr@gmail.com"
    <input
      className="mx-auto my-2"
      onBlur={[Function]}
      placeholder="email"
      type="email"
    />
    <input
      className="my-2"
      onChange={[Function]}
      type="password"
      value="notapassword"
    />
    <input
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
      value="Submit"
    />
  </form>
</Login>
`;
